tests/unit/Annotations/Annotation/GetArgumentCest.php

```diff
@@ -13,20 +13,45 @@
 
 namespace Phalcon\Test\Unit\Annotations\Annotation;
 
+use Phalcon\Annotations\Annotation;
 use UnitTester;
 
 class GetArgumentCest
 {
+    private $PHANNOT_T_STRING = 303;
     /**
      * Tests Phalcon\Annotations\Annotation :: getArgument()
      *
-     * @author Phalcon Team <team@phalcon.io>
-     * @since  2018-11-13
+     * @author Jeremy PASTOURET <https://github.com/jenovateurs>
+     * @since  2020-01-22
      */
     public function annotationsAnnotationGetArgument(UnitTester $I)
     {
         $I->wantToTest('Annotations\Annotation - getArgument()');
 
-        $I->skipTest('Need implementation');
+        $sValue = 'test';
+        $sValue1 = 'test1';
```

Comment:
@niden do we have something in our code standards regarding variable naming? Didn't see this Hungarian notation before in our Phalcon projects. 
@jenovateurs not saying this is wrong, just wanna check if there's a strict policy on this.

Reply:
We have the coding standard for Zephir and although we do not explicitly mention it, it kind of seeps into our tests i.e. camelCase, PSR-2 etc.

https://docs.phalcon.io/4.0/en/coding-standard

@jenovateurs lets change this to camelCase please just to be in sync with the rest of the tests. I will need to add something in the docs for this.

Reply:
@ruudboon and @niden, Okay, no problem, I will change it :-).